dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -85,6 +125,7 @@ const SiteSelectionForm = ({ filepath, data }: SiteSelectionForm) => {
         value={formData[currentPageId]}
         onFormValueChange={handleFormValueChange}
       />
+      {!!errors[currentPageId]?.length && <div>{errors[currentPageId][0]}</div>}
```

Comment:
Worth noting in the PR description that this displaying is only temporary since its just rendering the first error and with no styling where we should be iterating over them all.